mwdb/core/auth.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+import jwt
+
+from mwdb.core.config import app_config
+
+
+def generate_token(fields, scope, expiration=24 * 3600):
+    time_claims = {
+        "exp": datetime.datetime.now(tz=datetime.timezone.utc)
+        + datetime.timedelta(seconds=expiration),
+        "iat": datetime.datetime.now(tz=datetime.timezone.utc),
+    }
+    if "login" in fields.keys():
+        payload = {
+            **fields,
+            **time_claims,
+            "aud": app_config.mwdb.base_url,
+            "scope": scope,
+            "sub": fields["login"],
```

Comment:
I would rename `time_claims` to `token_claims` and move `aud` and `sub` claims there.

Let's also limit that conditional part
```
if "login" in fields.keys():
    payload["sub"] = fields["login"]
```